src/modules/DOMFunctions.js

```diff
@@ -34,14 +34,14 @@ export const tasklistAdd = (tasklist, value) => {
 
   tasklist.updateListHTML();
   updateLocalStorage(tasklist);
-}
+};
 
 // Set Handler for 'Clear All Completed' button
 export const clearAll = (tasklist) => {
   tasklist.clearAllCompleted();
   tasklist.updateListHTML();
   updateLocalStorage(tasklist);
-}
+};
 
 export const loadData = (tasklist) => {
   if (localStorage.datalist && localStorage.datalist.length > 0) {
```

Comment:
- [ ] Good job testing the `tasklistAdd` and the DOM remove button, however, you should create tests for all the pure functions responsible for adding and also deleting list items, that includes the clearAll remove tasks

Reply:
We have added a test for the `clearAll()` function.